res/css/views/rooms/_EventTile.pcss

```diff
@@ -403,7 +408,7 @@ $left-gutter: 64px;
         }
 
         &.mx_EventTile_continuation {
-            padding-top: 0px !important;
```

Comment:
`!important` has been required due to https://github.com/matrix-org/matrix-react-sdk/pull/9033/files#diff-5d3e25eb1d22dd7a96e0e87ea9c1298dc7dbce8eb965d30b2d1ebc1436bfec3fL484 inside `:not()`